daliuge-engine/dlg/apps/pyfunc.py

```diff
@@ -31,6 +31,7 @@
 import logging
 import os
 import pickle
+import pyext
```

Comment:
This module hasn't been updated for quite some time: 9 years according to https://github.com/refi64/PyExt.

It also looks like there's a version mismatch between PyPi and the master branch on the repository (Pypi is 0.6, repo is 0.8)? 

https://github.com/refi64/PyExt/blob/1e018b12752ceb279f11aee123ed773d63dcec7e/pyext.py#L25; https://pypi.org/project/pyext/#history.

It's such a small repository we could always fork it and use it within our code. J

Reply:
Agree! I think we should not even fork it, but just copy the part we are actually using here. Can still keep a proper reference to the source.